authors/apps/notifications/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Why do we have this?

Reply:
This is autogenerated when you create a new app, it enables you to register your models and manage them from the admin dashboard. Should I delete it?